stingray/tests/test_lightcurve.py

```diff
@@ -83,29 +84,29 @@ def test_input_countrate(self):
         assert np.allclose(lc.counts, np.zeros_like(countrate) +
                            mean_counts*dt)
 
-    @raises(TypeError)
     def test_init_with_none_data(self):
```

Comment:
I don't know is a good solution but I remember in a really big project I've worked on we ignore 501 pep8 error for test files.

And second thing we did  (and do in all projects I'm contributing into now) is set in pep8rc file line length to 120chars.